### ISSUE TITLE
There are no mathods to specify how fast turrets spin

### ISSUE THREAD

**Issue by Daiki9527** [Phobos-developers/Phobos#781]
### First check

- [X] The suggestion is **NOT** related to functionality of Ares, HAres, Kratos or any other YR engine extension.
- [X] The suggested feature / enhancement is **NOT** implemented in Ares/Phobos yet and the existing possibilites are not viable or good enough to achieve the wanted result.
- [X] I agree to elaborate the details if requested and provide thorough testing if the feature is implemented.
- [X] I added a very descriptive title to this issue.
- [X] I used the GitHub search and read the issue list to find a similar issue and didn't find it.
- [X] I thought out how the feature / enhancement should work and described it in the most detailed and clear way.

### Description

As mentioned on title.
Tried Ares's TurretROT, doesn't work.

### INI code

```ini
[SomeTechnoType]
TurretSpinRate=Value
```

**Comment by mevitar** [comment]
No, TurretROT works fine. TurretSpins=yes turrets will not use values higher than 5, no matter if you use ROT or TurretROT.

**Comment by Daiki9527** [comment]
> No, TurretROT works fine. TurretSpins=yes turrets will not use values higher than 5, no matter if you use ROT or TurretROT.

I was thinking how to make the turret faster so i used higher value. Your are right, what kind of rules is that.

**Comment by mevitar** [comment]
> Your are right, what kind of rules is that.

WestWood rules. :P
#blameWestWood
